src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -134,7 +139,7 @@ void AsyncRPCOperation_sendmany::main() {
         set_state(OperationStatus::FAILED);
     }
 
-    std::string s = strprintf("async rpc %s finished (status=%s", getId(), getStateAsString());
+    std::string s = strprintf("%s: finished (status=%s", getId(), getStateAsString());
     if (success) {
         s += strprintf(", tx=%s)\n", tx_.ToString());
```

Comment:
Or we just output the txid instead, since that should be sufficient to find the transaction in the local wallet.